lib/ocpp/v201/charge_point.cpp

```diff
@@ -2622,14 +2617,14 @@ void ChargePoint::handle_transaction_event_response(const EnhancedMessage<v201::
         evse_ids.push_back(original_msg.evse.value().id);
     } else {
         // add every evse_id with active transaction and given token
-        for (const auto& [evse_id, evse] : this->evses) {
-            const auto& transaction = evse->get_transaction();
-            if (transaction != nullptr and transaction->id_token.has_value()) {
-                if (transaction->id_token.value().idToken.get() == id_token.idToken.get()) {
-                    evse_ids.push_back(evse_id);
-                }
-            }
-        }
+        // for (const auto& [evse_id, evse] : this->evses) {
```

Comment:
Still not really sure what to do with this. It can probably be removed @Pietfried ?